docs/conf.py

```diff
@@ -32,12 +32,15 @@
 
 try:
     import astropy_helpers
-except ImportError:
+    astropy_helpers.sphinx
+except (ImportError, AttributeError):
     # Building from inside the docs/ directory?
     if os.path.basename(os.getcwd()) == 'docs':
         a_h_path = os.path.abspath(os.path.join('..', 'astropy_helpers'))
         if os.path.isdir(a_h_path):
             sys.path.insert(1, a_h_path)
+        if 'astropy_helpers' in sys.modules:
+            del sys.modules['astropy_helpers']
```

Comment:
This is to fix astropy_helpers import on Python 3, when not using `setup.py build_docs` (RTD uses sphinx-build directly).